src/Psl/DateTime/constants.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\DateTime;
+
+/**
+ * The number of nanoseconds in a microsecond.
+ *
+ * @var int
+ */
+const NANOSECONDS_PER_MICROSECOND = 1000;
+
+/**
+ * The number of nanoseconds in a millisecond.
+ *
+ * @var int
+ */
+const NANOSECONDS_PER_MILLISECOND = 1_000_000;
+
+/**
+ * The number of nanoseconds in a second.
+ *
+ * @var int
+ */
+const NANOSECONDS_PER_SECOND = 1_000_000_000;
+
+/**
+ * The number of microseconds in a millisecond.
+ *
+ * @var int
+ */
+const MICROSECONDS_PER_MILLISECOND = 1000;
+
+/**
+ * The number of microseconds in a second.
+ *
+ * @var int
+ */
+const MICROSECONDS_PER_SECOND = 1_000_000;
+
+/**
+ * The number of milliseconds in a second.
+ *
+ * @var int
+ */
+const MILLISECONDS_PER_SECOND = 1000;
+
+/**
+ * The number of seconds in a minute.
+ *
+ * @var int
+ */
+const SECONDS_PER_MINUTE = 60;
+
+/**
+ * The number of seconds in an hour.
+ *
+ * @var int
+ */
+const SECONDS_PER_HOUR = 3600;
+
+/**
+ * The number of seconds in a day.
+ *
+ * @var int
+ */
+const SECONDS_PER_DAY = 86_400;
+
+/**
+ * The number of seconds in a week.
+ *
+ * @var int
+ */
+const SECONDS_PER_WEEK = 604_800;
+
+/**
+ * The number of minutes in an hour.
+ *
+ * @var int
+ */
+const MINUTES_PER_HOUR = 60;
+
+/**
+ * The number of minutes in a day.
+ *
+ * @var int
+ */
+const MINUTES_PER_DAY = 1440;
+
+/**
+ * The number of minutes in a week.
+ *
+ * @var int
+ */
+const MINUTES_PER_WEEK = 10_080;
+
+/**
+ * The number of hours in a day.
+ *
+ * @var int
+ */
+const HOURS_PER_DAY = 24;
+
+/**
```

Comment:
maybe also

MONTHS_PER_YEAR

WEEKS_PER_YEAR is probably not possible due to 51-53 range?

Reply:
ADDED MONTHS_PER_YEAR, WEEKS_PER_YEAR is not static :/